Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:9.6-alpine
+FROM postgres:9.6-alpine AS base
```

Comment:
We use a newer version of Postgres now? Could this be updated to that version please.